dss/util/auth/authregistry.py

```diff
@@ -0,0 +1,15 @@
+import typing
+
+
+class AuthRegistry(type):
+
+    REGISTRY: typing.Dict[typing.Any, typing.Any] = {}
```

Comment:
Should this be `typing.Dict[str, typing.Any]`?

Reply:
derp, was testing something out thx.